prisma/schema.prisma

```diff
@@ -61,28 +66,32 @@ model Specialist {
 }
 
 model Organization {
-  id              String             @id @default(uuid()) @db.Uuid
-  createdAt       DateTime           @default(now())
-  name            String             @db.VarChar(128)
-  type            OrganizationType[]
-  yearsOnMarket   Int?               @db.SmallInt
-  formatOfWork    FormatOfWork?
-  addresses       Address[]
-  therapies       Therapy[]
-  isFreeReception Boolean?
-  description     String?            @db.Text
-  phone           String?            @db.VarChar(15)
-  email           String?            @db.VarChar(320)
-  website         String?            @db.Text
-  isActive        Boolean            @default(false)
-  instagram       String?            @db.Text
-  facebook        String?            @db.Text
-  youtube         String?            @db.Text
-  linkedin        String?            @db.Text
-  tiktok          String?            @db.Text
-  viber           String?            @db.Text
-  telegram        String?            @db.Text
-  searchEntry     SearchEntry?
+  id                    String             @id @default(uuid()) @db.Uuid
+  createdAt             DateTime           @default(now())
+  name                  String             @db.VarChar(128)
+  type                  OrganizationType[]
+  ownershipType         OwnershipType?
+  isInclusiveSpace      Boolean?
+  // this is light version of specialists relation to organization
+  expertSpecializations Specialization[]
+  yearsOnMarket         Int?               @db.SmallInt
```

Comment:
Would it make sense to replace it with `worksSince` and calculate `yearsOnMarket` dynamically?

Reply:
there were deeper discussion some time ago, related to case where specialist could work for some period, then make a break, then continue working...it is what it is for some time, and i would avoid touching this topic in scope of current pr if possible